docs/zoneinfo.rst

```diff
@@ -0,0 +1,370 @@
+:mod:`zoneinfo` --- IANA time zone support
+==========================================
+
+.. module:: zoneinfo
+    :synopsis: IANA time zone support
+
+.. versionadded:: 3.9
+
+.. moduleauthor:: Paul Ganssle <paul@ganssle.io>
+.. sectionauthor:: Paul Ganssle <paul@ganssle.io>
+
+--------------
+
+The :mod:`zoneinfo` module provides a concrete time zone implementation to
+support the IANA time zone database as originally specified in :pep:`615`. By
+default, :mod:`zoneinfo` uses the system's time zone data if available; if no
+system time zone data is available, the library will fall back to using the
+first-party ``tzdata`` package available on PyPI.
```

Comment:
Maybe it makes sense to specify a shared reference and link all the mentions like this?
```suggestion
first-party `tzdata`_ package available on PyPI.
```

```rst
.. _`tzdata`: https://pypi.org/project/tzdata/
```

Reply:
Hm, not sure how I feel about `tzdata` being normal text instead of monospace, but that does seem to be the style in other modules, so I've gone ahead and updated these.